test/test_metrics.py

```diff
@@ -94,7 +94,7 @@ def test_short_metrics_fallback_counter(self):
   def test_metrics_report(self):
     # TODO(jwtan): Add test to cover TrimIrGraph, SyncTensorsToData, TransferToServerAsync, IrValueTensorToXlaData
     xla_device = xm.xla_device()
-    t1 = torch.tensor(1456, device=xla_device)
+    t1 = torch.tensor(2077, device=xla_device)
```

Comment:
why this change?

Reply:
It looks like somehow we cache the device data in the server? If we reuse the same value for the scalar tensor, `TransferToServerTime` will be omitted in some of the tests.

Reply:
oh right.